app/views/staff/activities/comments.html.haml

```diff
@@ -27,15 +27,29 @@
             - if policy(:comment).create? && @report
               = a11y_action_link(t("page_content.comment.add"), new_activity_comment_path(@activity, report_id: @report.id))
 
-          %dl.govuk-summary-list.activity_comments
+          %ul.govuk-list.activity_comments
             - @comments.each do |comment|
               - report_presenter = ReportPresenter.new(comment.report)
-              .govuk-summary-list__row
-                %dt.govuk-summary-list__key
-                  = "#{report_presenter.financial_quarter_and_year} #{comment.report.description}"
-                %dd.govuk-summary-list__value
-                  = comment.comment
-                %dd.govuk-summary-list__actions
+              %li
+                %dl.govuk-summary-list
+                  .govuk-summary-list__row
+                    %dt.govuk-summary-list__key
+                      = Comment.model_name.human
+                    %dd.govuk-summary-list__value
+                      = comment.comment
                   - if policy(comment).update?
-                    = a11y_action_link(t("default.link.edit"), edit_activity_comment_path(@activity, comment), t("table.body.report.comment").downcase)
+                    .govuk-summary-list__row
+                      %dt.govuk-summary-list__key
+                      %dd.govuk-summary-list__value
+                        = a11y_action_link(t("page_content.comment.edit"), edit_activity_comment_path(@activity, comment))
+                  .govuk-summary-list__row
+                    %dt.govuk-summary-list__key
+                      Date
+                    %dd.govuk-summary-list__value
+                      = I18n.l(comment.created_at.to_date)
+                  .govuk-summary-list__row
+                    %dt.govuk-summary-list__key
+                      Comment reported in
+                    %dd.govuk-summary-list__value
+                      = link_to "#{report_presenter.financial_quarter_and_year} #{report_presenter.description}", report_path(comment.report)
```

Comment:
I'm not sure if we've covered these additions in an spec? Are we planning on adding some description of these additional comment properties to an existing feature spec? e.g. https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/65c9142273b0c4fcfbb04bf4042c21d0bb34dca6/spec/features/staff/users_can_create_a_comment_spec.rb#L51-L60